sdks/python/apache_beam/testing/benchmarks/nexmark/README.md

```diff
@@ -23,22 +23,28 @@
 
 For batch mode, a file needs to be generated first by running java suite and writing events to a file.
```

Comment:
Should we move this as well?

Reply:
I left it because I didn't think it was necessary to duplicate it for both variants.